liblwgeom/g_serialized.c

```diff
@@ -1173,8 +1173,8 @@ static size_t gserialized_from_gbox(const GBOX *gbox, uint8_t *buf)
 
 GSERIALIZED* gserialized_from_lwgeom(LWGEOM *geom, size_t *size)
 {
-	size_t expected_size = 0;
-	size_t return_size = 0;
+	uint32_t expected_size = 0;
```

Comment:
Sizes should remain as `size_t`, I think

Reply:
You are right.